common/assets/scss/utilities/_typography.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-secondary-text-colour {
```

Comment:
The only issue with using the `.govuk-` prefix here is that it suggests that the definition is coming from the GOV.UK Design System. 

The namespace pattern allows somebody to see which design system that definition exists in. I'm wondering if it's better to use the `.app-` prefix so its clear it is a definition within our local design system, not the GOV.UK one.

It could also lead to unintended side effects if, in later version they introduce the same class with more style definitions.